### ISSUE TITLE
References pages title

### ISSUE THREAD

**Issue by myrdn** [clic2000/clic2000.fr#9]
Pages `title` attribute should be unique. 

On references pages we should indicate pagination in titles to let users know they are on a different page.

`<title>Clic2000 - Références (page 1/3)</title>`

**Comment by mthld** [comment]
Taking care of this right now.

**Comment by mthld** [comment]
Changes live on staging: https://clic2000.github.io/clic2000.fr/references.html

Let's close the issue when it lands on production.
